angr/knowledge_plugins/propagations.py

```diff
@@ -0,0 +1,53 @@
+from .plugin import KnowledgeBasePlugin
+
+
+class Propagations(KnowledgeBasePlugin):
```

Comment:
I don't see where this new KB plugin is used. Is that because this PR is still WIP?

Reply:
Added WIP label